sourcegraph/sourcegraph
# [ally] notebook: fix arc toolkit issues

Closes #35670 
Closes #35678 
Closes #35681

## Test plan

- Verify checkmark icon is now hidden
- Verify notebook block is still tabbable and can navigate through blocks with arrow keys. 
- Verify notebook blocks now have the correct role (`article`) using VoiceOver and that they are listed in the Landmarks menu.
- Verify ARC Toolkit no long shows errors about aria-label on `div`s (since `div` was changed to `article` for notebook blocks)

## App preview:

- [Web](https://sg-web-jp-notebookarc.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-fdhzzaskwn.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
